examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -338,7 +345,18 @@ export class TaskList extends DataObject implements ITaskList {
 		}
 	};
 
-	protected async initializingFirstTime(): Promise<void> {
+	protected async initializingFirstTime(props: IBaseDocumentInitialState): Promise<void> {
+		const externalTaskListId = props?.externalTaskListId;
+		if (externalTaskListId === undefined) {
+			throw new Error(
+				"externalTaskListId not present in instantiation. Cannot instantiate task list",
+			);
+		}
+		this._externalTaskListId = externalTaskListId;
+		// TODO: remove console.log of externalTaskListId once it is used in upcoming PRs.
+		// Linter complains about it not being used and then tried to remove
+		// _externalTaskListId as well. This log is simpy to allow building for now.
+		console.log(this.externalTaskListId);
```

Comment:
I'll wait for the next PR to see the usage pattern, but maybe we can do away with the getter, since it is private, and just work with `this._externalTaskListId` (or remove the underscore if we don't need the getter). Just a thought.